src/main/java/com/es/phoneshop/web/MiniCartServlet.java

```diff
@@ -0,0 +1,29 @@
+package com.es.phoneshop.web;
+
+import com.es.phoneshop.model.cart.CartService;
+import com.es.phoneshop.model.cart.DefaultCartService;
+
+import javax.servlet.ServletConfig;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+
+
+public class MiniCartServlet extends HttpServlet {
+    protected static final String CART_JSP = "/WEB-INF/pages/miniCart.jsp";
```

Comment:
1) It's not cart jsp 2) It's good practice to make such String constants in each servlet, why it's done only here?